cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -358,9 +358,8 @@ runTxBuildCmd
                             , localNodeSocketPath = sockPath
                             }
 
-  AnyCardanoEra nodeEra
-    <- firstExceptT (ShelleyTxCmdQueryConvenienceError . AcqFailure)
-         . newExceptT $ determineEra cModeParams localNodeConnInfo
+  AnyCardanoEra nodeEra <- lift (determineEra cModeParams localNodeConnInfo)
+    & onLeft (throwE . ShelleyTxCmdQueryConvenienceError . AcqFailure)
```

Comment:
not a major deal but I'd like to stick with `left` in future

Reply:
I removed `throwE` from the commit history of the PR.